serverManager/service/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ if( RIALTO_ENABLE_CONFIG_FILE )
     add_compile_definitions(RIALTO_ENABLE_CONFIG_FILE)
     add_compile_definitions(RIALTO_CONFIG_PATH="${RIALTO_CONFIG_FILE_DIR}/rialto-config.json")
     add_compile_definitions(RIALTO_CONFIG_OVERRIDES_PATH="${RIALTO_CONFIG_OVERRIDES_FILE_DIR}/rialto-config-overrides.json")
+    add_compile_definitions(RIALTO_CONFIG_SOC_PATH="${RIALTO_CONFIG_FILE_DIR}/rialto_soc.json")
```

Comment:
Should this be ".../rialto-soc.json" (dash not underscore) to be consistent with the file naming conventions on lines 69 & 70

Reply:
it can't be. Comcast decided to have rialto_soc.json